### DISCUSSION TITLE
when create a dataset use a string a error raise

### DISCUSSION THREAD

**Discussion by Baozi2** [jeremyevans/sequel#2203] [General]
when i create a dataset example: DB['goods_20240814']  a error: PG::SyntaxError: ERROR:  syntax error at or near,
when i change to DB[:goods_20240814] , it is right

**Comment by jeremyevans** [comment]
This is expected, as strings are treated as raw SQL: http://sequel.jeremyevans.net/rdoc/classes/Sequel/Database.html#method-i-5B-5D

**Reply by Baozi2** [reply]
got it , thank you!
